book_source/03_intermediate_users_guide/01_pecan_xml.Rmd

```diff
@@ -249,163 +542,29 @@ Only if this section is defined a sensitivity analysis is done. This section wil
-
-As of version 1.3.7 the `<name>` tag has been renamed `<type>`. The `<type>` tag refers to the "type" of model and is used for a) identifying appropriate pfts and b) identifying whether appropriate inputs (i.e. met data) are available for the given site and model (or if they can be created). 
-
-To ensure compatability, the code will automatically convert from `<name>` to `<type>`.
+*  `variable` : [optional] name of one (or more) variables the analysis should be run for. If not specified, sensitivity.analysis variable is used, otherwise default is GPP.
+* `perpft` : [optional] if `TRUE` a sensitivity analysis on PFT-specific outputs will be run. This is only possible if your model provides PFT-specific outputs for the `variable` requested. This tag only affects the output processing, not the number of samples proposed for the analysis nor the model execution.
 
-#### functions: 
-
-`write.configs()`, `run.models()`
-
-#### tags
-
-```xml
-<model>
-	<id>7</id>
-	<type>ED2</type>
-	<binary>/usr/local/bin/ed2.r82</binary>
-	<job.sh>module load hdf5</job.sh>
-	<config.header>
-		<!--...xml code passed directly to config file...-->
-	</config.header>
-</model>
-```
-
-* **id** : [optional/required] id in the models database table, see above.  
-* **name** : **OBSOLETE** name of the model, use type from version 1.3.7
-* **type** : [optional/required] type of model, see above.
-* **binary** : [optional/required] path to the model executable, see above.  
-* **job.sh** : [optional] additional options to add to the job.sh at the top  
-* **config.headers** : [optional] XML that will appear at the start of generated config files.
-
-#### ED2 specific tags
-
-Following variables are ED specific and are used in the [ED2 Configuration](ED2-Configuration).
-
-Starting at 1.3.7 the tags for inputs have moved to `<run><inputs>`. This includes, veg, soil, psscss, inputs.
-
-```xml
-	<edin>/home/carya/runs/PEcAn_4/ED2IN.template</edin>
-	<config.header>
-		<radiation>
-			<lai_min>0.01</lai_min>
-		</radiation>
-		<ed_misc>
-			<output_month>12</output_month>      
-		</ed_misc> 
-	</config.header>
-	<phenol.scheme>0</phenol.scheme>
-```
+This information is currently used by the following PEcAn workflow functions:
 
-  
-* **edin** : [required] template used to write ED2IN file
-* **veg** : **OBSOLETE** [required] location of VEG database, now part of `<run><inputs>
-* **soil** : **OBSOLETE** [required] location of soild database, now part of `<run><inputs>
-* **psscss** : **OBSOLETE** [required] location of site inforation, now part of `<run><inputs>`. Should be specified as `<pss>`, `<css>` and `<site>`.
-* **inputs** : **OBSOLETE** [required] location of additional input files (e.g. data assimilation data), now part of `<run><inputs>`. Should be specified as `<lu>` and `<thsums>`.
+- `write.configs()` - ???
+- `run.sensitivity.analysis()` - ???
```

Comment:
why always ???

Reply:
Because I wanted to include a line of documentation for these functions, but had not gotten around to it. Working on adding that now.

Reply:
Addressed in 1bacb771a.